PRDs/20251215-shell-completion/PRD.md

```
@@ -0,0 +1,653 @@
+Completions will be automatically loaded in new terminal sessions.
+
+**Option 2: System-wide installation (requires root)**
+
+```bash
+# Generate and install for all users
+sudo metaschema-cli shell-completion bash --to /etc/bash_completion.d/metaschema-cli
+
+# Reload completions
+source /etc/bash_completion.d/metaschema-cli
+```
+
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)